client/src/components/web-extension/ImportWallet/ImportWalletPage.jsx

```diff
@@ -0,0 +1,82 @@
+import React, { useCallback } from 'react';
+import { useDispatch } from 'react-redux';
+import { Button, Form as FormBS } from 'react-bootstrap';
+import { useSelector } from 'react-redux';
+import { Formik, Form } from 'formik';
+import { KeyFactory } from 'casper-storage';
+
+import { selectCreateWalletTotalKeywords } from "@cd/selectors/createWallet";
+import { setNextStep, updateKeyphrase } from "@cd/actions/createWalletActions";
+
+import FieldKeyphrase from "./FieldKeyphrase";
+
+const ImportWallet = () => {
+	const dispatch = useDispatch();
+	const TOTAL_KEYWORDS = useSelector(selectCreateWalletTotalKeywords);
+
+	const onValidate = useCallback(values => {
+		const { keyphrase } = values;
+		const errors = {};
+
+		/**
+		 * Make sure keyphrase has 12 words
+		 */
+		if (keyphrase.some(key => key === '')) {
+			errors.keyphrase = 'Keypharse does not empty';
+
+			return errors;
+		}
+
+		/**
+		 * Make sure keyphrase is valid with `KeyFactory.getInstance().validate(keyphrase)`
+		 */
+		if (!KeyFactory.getInstance().validate(keyphrase.join(' '))) {
+			errors.keyphrase = 'Keypharse is not validated';
```

Comment:
Please change this error message to `**Keyphrase is invalid. Please recheck your keyphrase.**`